packages/e2e-tests/README.md

```diff
@@ -0,0 +1,29 @@
+# End to end tests
+
+## Install playwright
+
+cd into packages/e2e-tests
+
+and run `pnpm create playwright`
+
+in the upcoming dialog
+
+selects "tests" as target for tests and skip the creation of playwright.config.ts
+
+after the install remove the created tests/examples and packages/e2e-tests/tests/example.spec.ts
```

Comment:
I doubt that this is a good solution, won't `npx playwright install --with-deps` suffice?